config/pitest-suppressions/pitest-tree-walker-suppressions.xml

```diff
@@ -1,11 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<suppressedMutations>
-  <mutation unstable="false">
```

Comment:
please keep file but remove "<mutation"  only, in this case it will generate good diff in next PRs when author cause more mutants to appear.